code/game/objects/effects/spiderwebs.dm

```diff
@@ -128,3 +160,31 @@
 	for(var/atom/movable/A in contents)
 		A.forceMove(T)
 	return ..()
+
+/obj/structure/spider/sticky
+	name = "sticky web"
+	icon = 'icons/effects/effects.dmi'
+	desc = "extremly sticky soft silk ."
+	icon_state = "verystickyweb"
+	max_integrity = 20
+
+/obj/structure/spider/sticky/CanAllowThrough(atom/movable/mover, border_dir)
+	. = ..()
+	if(isspider(mover))
+		return TRUE
+	else if(isliving(mover))
+		if(istype(mover.pulledby, /mob/living/basic/spiderling))
+			return TRUE
+		balloon_alert(mover, "stuck in web!")
+		return FALSE
```

Comment:
```suggestion
	if(isspider(mover))
		return TRUE
	if(!isliving(mover))
		return
	if(isspider(mover.pulledby))
		return TRUE
	balloon_alert(mover, "stuck in web!")
	return FALSE
```

Few things here:
- You don't need an else following a return.
- You should "early return" whenever possible, basically meaning that if the rest of your proc would be indented inside an if statement then instead just invert the if statement and return from that point.
- You're checking if something is being dragged by a spiderling again?